apps/researcher/src/app/[locale]/object/[id]/page.tsx

```diff
@@ -88,6 +88,13 @@ export default async function Details({params}: Props) {
     return <div data-testid="no-object">{t('noObject')}</div>;
   }
 
+  const imagesWithAlt =
```

Comment:
Minor: is it necessary to say 'withAlt'? If we, for some reason, decide to remove the alt, we also have to rename the variable. Is it possible to use a more persistent name?